Gordon360/Controllers/RecIM/TeamsController.cs

```diff
@@ -202,7 +202,7 @@ public async Task<ActionResult<TeamInviteViewModel>> AcceptTeamInvite(int teamID
             if (invite is null)
                 return NotFound("You were not invited by this team.");
             if (username != invite.ParticipantUsername)
-                return Forbid($"You are not permitted to accept invitations for {invite.ParticipantUsername}.");
+                return Forbid($"You are not permitted to accept invitations for another participant.");
```

Comment:
The reason why I changed this was we should not display the user that the invite was intended for to the user that is trying to force them to accept.